cmd/cbft/init_mem.go

```diff
@@ -33,6 +36,13 @@ func initMemOptions(options map[string]string) (err error) {
 		}
 		memQuota = uint64(fmq)
 	}
+	memoryLimit, err := getMemoryLimit()
+	if memoryLimit < memQuota {
+		memQuota = uint64(MemoryQuotaThreshold * float32(memoryLimit))
+		// multiplying by the threshold avoids the OOM error by having (1-threshold)*memoryLimit
+		// as a buffer.
+	}
```

Comment:
Consider CfgSetOptions here()?
Since this is for container level limits(which are unique to each mgr instance), these aren't applicable to the other nodes in the cluster. So setting it locally should be enough?